apps/condo/domains/common/hooks/useValidations.ts

```diff
@@ -134,6 +137,22 @@ export const useValidations: UseValidations = (settings = {}) => {
             }
         }
 
+    const addressValidator: (selectedPropertyId, isMatchSelectedPropertyAndInputPropertyName) => Rule =
+        (selectedPropertyId, isMatchSelectedPropertyAndInputPropertyName) => {
+            return {
+                validator: (_, value) => {
+                    const searchValueLength = get(value, 'length', 0)
+                    if (searchValueLength === 0
```

Comment:
this way it would be quite a bit more redable

```
const isPropertyMatched = selectedPropertyId !== undefined && isMatchSelectedPropertyAndInputPropertyName

if (searchValueLength < 1 || isPropertyMatched) return Promise.resolve()

return Promise.reject(AddressNotSelected)
```